tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -958,6 +958,12 @@ show_techsupport/test_auto_techsupport.py:
     conditions:
       - (is_multi_asic==True)
 
+show_techsupport/test_auto_techsupport.py::TestAutoTechSupport::test_max_limit[core]:
+  skip:
```

Comment:
Can we use xfail instead? Since it is a "might fail" instead of 100% failure?

Reply:
@yxieca updated to xfail